massa-execution-worker/src/context.rs

```diff
@@ -126,6 +127,9 @@ pub struct ExecutionContext {
 
     /// operation id that originally caused this execution (if any)
     pub origin_operation_id: Option<OperationId>,
+
+    // cache of pre compiled sc modules
```

Comment:
I'm picky but this is not "pre compiled", sc are compiled into assembly (like x86_64) by wasmer

Reply:
It was "pre compiled" as in previously compiled not literal pre compilation, I can understand what that was misleading I updated it